src/Users/README.md

```diff
@@ -61,16 +61,63 @@ if (!$result) {
 }
 ```
 
+### UPDATE OWN BASIC INFO
+```php
+$updatedUser = \ATDev\RocketChat\Users\User::updateOwnBasicInfo([
+    'email' => 'test@updated.com',
+    'name' => 'Updated Name',
+    'username' => 'new-username',
+    'currentPassword' => hash('sha256', 'q1w2e3r4t5'),
+    'newPassword' => 'newPassw0rd',
+    'customFields' => [
+        'twitter' => '@example'
+    ]
+]);
+if (!$updatedUser) {
+    $error = \ATDev\RocketChat\Users\User::getError();
+}
+```
+
+### SET ACTIVE STATUS
+```php
+$user = new \ATDev\RocketChat\Users\User("[USER ID]");
+$statusResult = $user->setActiveStatus(true);
+if (!$statusResult) {
+    $statusResult->getError();
+}
+```
+
+### SET STATUS MESSAGE
+```php
+$result = \ATDev\RocketChat\Users\User::setStatus("New status message", "away");
+if (!$result) {
+    $error = \ATDev\RocketChat\Users\User::getError();
+}
+```
+
+### GET STATUS
+```php
+$user = new \ATDev\RocketChat\Users\User("[USER ID]");
+// or by username
+$user = (new \ATDev\RocketChat\Users\User())->setUsername("[USERNAME]");
+$user = \ATDev\RocketChat\Users\User::getStatus($user);
```

Comment:
This should be non-static for sure. We already created the instance, and should call its method.